ui/lib/replication/addon/components/enable-replication-form.hbs

```diff
@@ -0,0 +1,186 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<form {{on "submit" (fn this.onSubmit this.data)}} data-test-replication-enable-form>
+  <MessageError @errorMessage={{this.error}} />
+
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <label for="replication-mode" class="is-label">
+      Cluster mode
+    </label>
+    <div class="field is-expanded">
+      <div class="control select is-fullwidth">
+        <select
+          onchange={{action (mut this.data.mode) value="target.value"}}
```

Comment:
Is updating this to the `{{on "change"....}}` syntax something we should tackle in glimmer PRs? I understand not wanting to in this case since the glimmer update was already quite a few changes

Reply:
But if we refactor it would mean we no longer get to use `value` [docs](https://guides.emberjs.com/release/upgrading/current-edition/action-on-and-fn/#toc_the-on-modifier)

Reply:
good catch! I did update the `input`s on this page (using the `Input` component helper i no longer best practice due to two-way binding) so it would be fairly simple to update all the inputs to use the same onChange method